test/run/conversions.mo

```diff
@@ -3,14 +3,14 @@ import Prim "mo:⛔";
 // Nat <--> Nat8
 
 func test_Nat_Nat8(n1 : Nat, n2 : Nat8) {
-  assert(Prim.natToNat8 n1 == n2);
-  assert(Prim.intToNat8Wrap n1 == n2);
-  assert(n1 == Prim.nat8ToNat n2);
+  assert (Prim.natToNat8 n1 == n2);
```

Comment:
`assert` is syntax, not a function, so the parens are redundant. If you feel like it :-)

Reply:
Maybe it makes sense to add this rule to the formatter? @rvanasa 

Also, I noticed there's a lot of formatting discrepancies in the motoko repo, since they haven't been ran through the formatter. Maybe it makes sense to format the whole repo and require it in CI (as long as the tool can be used by non-vscode users). 

I only mention this because the formatting changes clutters the `Files Changed` for PR reviews.